sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/storage/blob/AzureStorageBlobAutoConfiguration.java

```diff
@@ -31,14 +31,13 @@
 @ConditionalOnAnyProperty(prefix = "spring.cloud.azure.storage.blob", name = { "account-name", "endpoint", "connection-string" })
 public class AzureStorageBlobAutoConfiguration extends AzureServiceConfigurationBase {
 
-    public AzureStorageBlobAutoConfiguration(AzureGlobalProperties azureGlobalProperties) {
-        super(azureGlobalProperties);
+    public AzureStorageBlobAutoConfiguration() {
     }
 
     @Bean
     @ConfigurationProperties(AzureStorageBlobProperties.PREFIX)
-    public AzureStorageBlobProperties azureStorageBlobProperties() {
-        return loadProperties(this.azureGlobalProperties, new AzureStorageBlobProperties());
+    public AzureStorageBlobProperties azureStorageBlobProperties(AzureGlobalProperties azureGlobalProperties) {
+        return loadProperties(azureGlobalProperties, new AzureStorageBlobProperties());
     }
```

Comment:
Why are we changing to use this kind of injection?

Reply:
I encounterd a strange problem at that time, but it  seems is gone now, I have reverted the code here.